docs/INSTALL.md

```diff
@@ -8,21 +8,21 @@
 ## Linux
 
   ### Install **OpenVINO™ integration with TensorFlow** PyPi release
-  * Includes pre-built libraries of OpenVINO™ version 2022.1.0. The users do not have to install OpenVINO™ separately 
+  * Includes pre-built libraries of OpenVINO™ version 2022.2.0. The users do not have to install OpenVINO™ separately 
   * Supports Intel<sup>®</sup> CPUs, Intel<sup>®</sup> integrated GPUs, and Intel<sup>®</sup> Movidius™ Vision Processing Units (VPUs). No VAD-M support 
 
         pip3 install -U pip
-        pip3 install tensorflow==2.9.1
-        pip3 install openvino-tensorflow==2.1.0
-    The openvino-tensorflow PyPi package is cross-compatible with PATCH versions of TensorFlow. For example, openvino-tensorflow wheel for TF 2.9.1 would work with any future PATCH versions like TF 2.9.2, and 2.9.3
+        pip3 install tensorflow==2.9.2
+        pip3 install openvino-tensorflow==2.2.0
+    The openvino-tensorflow PyPi package is cross-compatible with PATCH versions of TensorFlow. For example, openvino-tensorflow wheel for TF 2.9.2 would work with any future PATCH versions like TF 2.9.2, and 2.9.3
   <br/>  
  
   ### Install **OpenVINO™ integration with TensorFlow** PyPi release alongside the Intel® Distribution of OpenVINO™ Toolkit for VAD-M Support
-  * Compatible with OpenVINO™ version 2022.1.0
+  * Compatible with OpenVINO™ version 2022.2.0
   * Supports Intel<sup>®</sup> Vision Accelerator Design with Movidius™ (VAD-M), it also supports Intel<sup>®</sup> CPUs, Intel<sup>®</sup> integrated GPUs and Intel<sup>®</sup> Movidius™ Vision Processing Units (VPUs)
   * To use it:
     1. Install tensorflow and openvino-tensorflow packages from PyPi as explained in the section above
-    2. Download & install Intel® Distribution of OpenVINO™ Toolkit 2022.1.0 release along with its dependencies from ([https://software.intel.com/en-us/openvino-toolkit/download](https://software.intel.com/content/www/us/en/develop/tools/openvino-toolkit/download.html)).
+    2. Download & install Intel® Distribution of OpenVINO™ Toolkit 2022.2.0 release along with its dependencies from ([https://software.intel.com/en-us/openvino-toolkit/download](https://software.intel.com/content/www/us/en/develop/tools/openvino-toolkit/download.html)).
     3. Initialize the OpenVINO™ environment by running the `setupvars.sh` located in <code>\<openvino\_install\_directory\>\/bin</code> using the command below. This step needs to be executed in the same environment which is used for the TensorFlow model inference using openvino-tensorflow.
```

Comment:
We need to change this too, now setupvars.sh is present in <openvino_root_directory> rather than <openvino_root_directory>/bin, bin directory is removed from openvino_2022.1.0.

Reply:
Will take this change as a part of another docs PR